DQM/Integration/python/clients/dt4ml_dqm_sourceclient-live_cfg.py

```diff
@@ -117,6 +118,7 @@
     process.twinMuxStage2Digis.DTTM7_FED_Source = "rawDataRepacker"
     process.dtunpacker.inputLabel = "rawDataRepacker"
     process.gtDigis.DaqGtInputTag = "rawDataRepacker"
+    process.gtStage2Digis.InputLabel = "rawDataCollector"
```

Comment:
Shouldn't it be here:
```PYTHON
    process.gtStage2Digis.InputLabel = "rawDataRepacker"
```

Reply:
I think it can be whatever. It is not in any path apparently.